### PR TITLE
Migration material components 1.1.0

### PR DESCRIPTION
**Pull request by mlcsouza**
In this PR, the Android Support Design Library was replaced by Material Components.
All components were migrated to AndroidX correspondence

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -47,4 +47,5 @@ ext {
     androidxJunitVersion = "1.1.1"
     truthVersion = '1.2.0'
     robolectricVersion = '4.3'
+    material = '1.1.0'
```
Comment (pmunizferreira) [inline build.gradle:50]: que tal mantermos o padrao e renomearmos para `materialVersion`? :/

Comment (pmunizferreira) [reply build.gradle:50]: `supportLibraryVersion` ainda esta em uso?

File: designsystem/src/main/kotlin/com/natura/android/button/DsPrimaryButton.kt [hunk 2]
```diff
@@ -2,11 +2,11 @@ package com.natura.android.button
...
 import android.annotation.SuppressLint
 import android.content.Context
-import android.support.constraint.ConstraintLayout
-import android.support.v4.content.ContextCompat
-import android.support.v7.widget.AppCompatTextView
 import android.util.AttributeSet
 import android.view.View
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
```
Comment (pmunizferreira) [inline designsystem/src/main/kotlin/com/natura/android/button/DsPrimaryButton.kt:9]: existe um card pra removermos esse componente. Ele nao esta em uso e foi adicionado aqui sem revisao. Lucas solicitou sua remocao.
Alem disso podemos usar o nativo do android.

File: designsystem/src/main/res/layout/ds_default_error.xml [hunk 3]
```diff
@@ -39,22 +40,22 @@
...
-    <com.natura.android.button.DsPrimaryButton
+    <com.google.android.material.button.MaterialButton
         android:id="@+id/ds_default_error_button"
-        android:layout_width="0dp"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
         android:layout_marginBottom="@dimen/ds_spacing_little"
+        android:text="RECARERGAR"
```
Comment (viniciusponciano-tw) [inline designsystem/src/main/res/layout/ds_default_error.xml:56]: A label está incorreta o certo seria RECARREGAR

Comment (mlcsouza) [reply designsystem/src/main/res/layout/ds_default_error.xml:56]: Bem observado!

**Comment by mlcsouza** [comment]
# Description
- Update on core components lib from Android Support Design V28 to Material Android Components from Android V1.1.0
- We added more than 70 new color references in the lib
- We added Natura, Avon and The Body Shop color themes in light and dark mode

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Pull request merged**